mito-ai/src/Extensions/AiChat/ChatMessage/AssistantCodeBlock.tsx

```diff
@@ -57,7 +56,7 @@ const AssistantCodeBlock: React.FC<IAssistantCodeBlockProps> = ({
                 >
                     <span className="agent-mode-toggle-content">
                         <CodeIcon />
-                        Generated {lineCount} lines of code
+                        {codeSummary ?? 'Generated code'}
                     </span>
                     <ExpandIcon isExpanded={isCodeExpanded} />
                 </div>
```

Comment:
Can we update this slightly so that if the text goes over one like (which it def will sometimes) that the size of the CodeIcon and ExpandIcon don't shrink. 
<img width="336" alt="Screenshot 2025-07-05 at 9 03 31 PM" src="https://github.com/user-attachments/assets/1fef3865-c3ae-4595-89bb-3dc34706440d" />